src/js/page/ui/main-menu.js

```diff
@@ -63,7 +63,7 @@ class MainMenu extends (require('events').EventEmitter) {
   }
 
   _onTextInputChange(event) {
-    var val = this._pasteInput.value.trim();
+    var val = this._pasteInput.value.trim().replace(/\0$/, "");
```

Comment:
FYI, according to this StackOverflow post, this will replace only one instance of a null terminator (not multiple): https://stackoverflow.com/a/22809513/844882

Reply:
```suggestion
    var val = this._pasteInput.value.trim().replace(/\0+$/g, "");
```
I suggest adding `+` to catch sequential nulls and `/g` to make the search global.